src/desktop/apps/authentication/components/ModalContainer.tsx

```diff
@@ -97,7 +97,7 @@ export class ModalContainer extends React.Component<any> {
           facebook: sd.AP.facebookPath,
           twitter: sd.AP.twitterPath,
         }}
-        csrf={sd.CSRF_TOKEN}
+        csrf={Cookies && Cookies.get && Cookies.get("CSRF_TOKEN")}
```

Comment:
If cookies is required in this file, why do we need to check for `Cookies && Cookies.get`?

Reply:
This had me stumped for a while, in short `cookies-js` isn't isomorphic. When required, `cookies-js` returns a different type depending on if it was run in a browser or node. 

In the browser it's a valid object with a `get` method. 
In node it's a factory function that expects a window-like object such as `JSDOM`.

It might be worth investigating what it would take to replace this library with one more widely supported.

https://github.com/ScottHamper/Cookies/blob/master/src/cookies.js#L156